src/home.js

```diff
@@ -0,0 +1,147 @@
+};
+
+const likeIcon = () => {
+  const likeButton = document.querySelectorAll('button');
+  likeButton.forEach((button) => {
+    button.addEventListener('click', () => {
+      addlike(button);
+    });
+  });
+};
+
+const pokemonShow = async () => {
+  const pokemons = await likesShow();
+  container.innerHTML = '';
+  for (let i = 0; i < pokemons.length; i += 1) {
+    container.innerHTML += `<article class="pokemon-card">
+        <img src="${pokemons[i].img}" alt="${
+  pokemons[i].name
+}" class="pokemon-img"/>
+        <h2 class="pokemon-name">${pokemons[i].name.toUpperCase()}</h2>
+        <button class="like-button" id="${pokemons[i].id}">
+      <svg
+        xmlns="http://www.w3.org/2000/svg"
+        xmlns:xlink="http://www.w3.org/1999/xlink"
+        version="1.1"
+        width="256"
+        height="256"
+        viewBox="0 0 256 256"
+        xml:space="preserve"
+        class="like-icon"
+      >
+        <desc>Created with Fabric.js 1.7.22</desc>
+        <defs></defs>
+        <g transform="translate(128 128) scale(0.72 0.72)" style="">
+          <g
+            style="
+              stroke: none;
+              stroke-width: 0;
+              stroke-dasharray: none;
+              stroke-linecap: butt;
+              stroke-linejoin: miter;
+              stroke-miterlimit: 10;
+              fill: none;
+              fill-rule: nonzero;
+              opacity: 1;
+            "
+            transform="translate(-175.05 -175.05000000000004) scale(3.89 3.89)"
+          >
+            <path
+              d="M 45 86.304 c -0.459 0 -0.919 -0.105 -1.342 -0.316 C 12.259 70.287 0.527 47.557 0.019 31.52 C -0.325 20.69 4.171 11.488 12.045 6.905 C 20.918 1.743 32.568 2.841 45 9.968 c 12.432 -7.127 24.085 -8.225 32.955 -3.063 c 7.874 4.583 12.369 13.785 12.026 24.615 c -0.509 16.037 -12.24 38.768 -43.64 54.468 C 45.919 86.198 45.46 86.304 45 86.304 z M 24.426 9.698 c -3.46 0 -6.615 0.793 -9.363 2.393 c -6.026 3.507 -9.324 10.52 -9.047 19.239 C 6.462 45.415 16.968 65.466 45 79.939 C 73.032 65.466 83.537 45.415 83.984 31.33 c 0.276 -8.719 -3.021 -15.732 -9.048 -19.239 c -7.25 -4.219 -17.325 -2.825 -28.373 3.924 c -0.959 0.587 -2.167 0.587 -3.127 0 C 36.577 11.824 30.092 9.698 24.426 9.698 z"
+              style="
+                stroke: none;
+                stroke-width: 1;
+                stroke-dasharray: none;
+                stroke-linecap: butt;
+                stroke-linejoin: miter;
+                stroke-miterlimit: 10;
+                fill: rgb(0, 0, 0);
+                fill-rule: nonzero;
+                opacity: 1;
+              "
+              transform=" matrix(1 0 0 1 0 0) "
+              stroke-linecap="round"
+            />
+          </g>
+        </g>
+      </svg>
+    </button>
+        <p class="likes-counter" id="${pokemons[i].id}counter">${
+  pokemons[i].likes || 0
+} likes</p>
+        <a href="#" class="comment" id="${
+  pokemons[i].id
+}">Comment</a></article>`;
```

Comment:
- [ ] Please add "s" to "Comment" like in the design. 
![s1](https://user-images.githubusercontent.com/63972670/169469803-ab39ce20-4d94-40dc-925a-ee0d3d0e97fc.JPG)